ui/src/app/views/cohort-builder/search/search-group/search-group.component.css

```diff
@@ -1,3 +1,3 @@
 .bg-count {
     background-color: #d7d7d7;
-}
+}
```

Comment:
These little editor inconsistencies are annoying in PRs. Since GitHub seems to prefer ending in a newline, can we agree to configure our editors to do the same?

Reply:
Sure

Reply:
sure.